modules/View.js

```diff
@@ -78,6 +79,28 @@
       return this.viewState.set.apply(this.viewState, arguments);
     },
 
+    _initializeBehaviors: function() {
+      var self = this;
+      if (!_.isEmpty(this.behaviors)) {
+        self.__behaviorInstances = {};
+        _.each(this.behaviors, function(behaviorDefinition, alias) {
+          var BehaviorClass = behaviorDefinition.behavior;
+          var behaviorOptions = _.pick(behaviorDefinition, function(value, key) {
+            return key !== 'behavior';
+          });
+          // Is it okay to override potential 'view' parameters?
+          behaviorOptions.view = self;
+          self.__behaviorInstances[alias] = new BehaviorClass(behaviorOptions);
```

Comment:
Should throw a more specific error message here if the Behavior definition is wrong.